src/osd/OSD.cc

```diff
@@ -1965,7 +2000,15 @@ int OSD::init()
   peering_wq.drain();
 
   dout(0) << "done with init, starting boot process" << dendl;
-  set_state(STATE_BOOTING);
+
+  // subscribe to any pg creations
+  monc->sub_want("osd_pg_creates", last_pg_create_epoch, 0);
+
+  // we don't need to ask for an osdmap here; objecter will
+  //monc->sub_want("osdmap", osdmap->get_epoch(), CEPH_SUBSCRIBE_ONETIME);
```

Comment:
I tried that first but had trouble getting it to send a single subscribe instead of multiples.  But it might be possible later.. hence leaving the ugly comment in the code.